Automattic/jetpack
# Search: Better handling of Unlimited API requests.

Follow-up PR addressing "0/Unlimited" display in usage meters.

#### Changes proposed in this Pull Request:


Better handles the case where the API returns a huge request limit. Was showing "0/Unlimited". Now shows the request count from the last month if available (245/Unlimited) or just "Unlimited" if no request count was provided.

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






1. Visit new Search dashboard.
2. Check that limits are shown correctly under the "Search requests" meter.
3. Use Dev Tools to update the values.
4. `count === 0` --> "Unlimited"
5. `count !==0` --> "count/Unlimited"
6. Normal request limit from API --> "count/limit"

Screenshot:

![SnapUnlimited](https://user-images.githubusercontent.com/40267301/195595208-ab6d783e-681a-4376-8655-d2f46971c7d2.png)
